### PR TITLE
pull in kitty-items-web

### PR DESCRIPTION
**Pull request by orodio**

### CODE REVIEW

**Review by jordankitty — APPROVED** [review]
🚀

File: kitty-items-web/src/flow/fetch-market-item.script.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import * as fcl from "@onflow/fcl"
+import * as t from "@onflow/types"
+// import {batch} from "./util/batch"
```
Comment (orodio) [inline kitty-items-web/src/flow/fetch-market-item.script.js:3]: This is supposed to be commented out. The commented out code is the code we want to be using, but can't yet do to contract stuff.

**Review by psiemens — APPROVED** [review]
LGTM.

Does it make sense to link to the new FCL quickstart from this README?

**Comment by orodio** [comment]
Yeah, ill have another PR shortly for that.

**Pull request merged**